{{ cookiecutter.__dirname }}/.github/workflows/build.yml

```diff
@@ -54,6 +54,22 @@ jobs:
         pre-commit run --all-files
     - name: Build assets
       run: circuitpython-build-bundles --filename_prefix {% raw %}${{ steps.repo-name.outputs.repo-name }}{% endraw -%} {% if cookiecutter.target_bundle == 'Community' %} --package_folder_prefix {% if cookiecutter.library_prefix -%} {{ cookiecutter.library_prefix | lower | replace(' ', '_') }}_ {%- else %}""{%- endif -%}{%- endif %} --library_location .
+    - name: Check Sizes
+      run: |
+        git clone https://github.com/circuitpython/CircuitPython_Org_size_tools.git
```

Comment:
You could probably use `actions/checkout` for this

Reply:
latest commit uses this checkout action instead of `git clone`. Tested over on this branch: https://github.com/FoamyGuy/Adafruit_CircuitPython_SI1145/pull/1